### PR TITLE
Fixes to split-lazy

### PR DESCRIPTION
**Pull request by wgb14**
Add prefix and start from 1: change from `0.jsonl.gz` to `cuts_XL_raw.1.jsonl.gz`

However, it's hard to generate `cuts_XL_raw.0001.jsonl.gz`, because we cannot know how many splits would be in advance while doing lazy-loading.

### CODE REVIEW

File: lhotse/audio.py [hunk 1]
```diff
@@ -933,7 +933,9 @@ def split(
             )
         ]
 
-    def split_lazy(self, output_dir: Pathlike, chunk_size: int) -> List["RecordingSet"]:
+    def split_lazy(
+        self, output_dir: Pathlike, chunk_size: int, prefix: str
```
Comment (pzelasko) [inline lhotse/audio.py:937]:
```suggestion
        self, output_dir: Pathlike, chunk_size: int, prefix: str = ""
```

It seems like prefix could be optional here (and in other functions as well)

Comment (wgb14) [reply lhotse/audio.py:937]: Yes, it looks good to have a default value. But if `prefix=""`, `(output_dir / prefix)` doesn't work, what about setting default `prefix="split"`?

Comment (wgb14) [reply lhotse/audio.py]: Fixed

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**